tests/conftest.py

```diff
@@ -54,6 +54,20 @@ def nowin():
         pytest.skip("Disabled on Windows")
 
 
+
+@pytest.fixture(scope="session")
+def tol():
+    """
+    This fixture returns a tolerance to compare floats
+    on a particular operating system. The reason we sometimes use
+    OS specific tolerance is that some platforms don't have a proper
+    long double type, resulting in a loss of precision
+    when fread converts double literals into double numbers.
+    """
+    tols = {"win32": 1e-15}
+    return tols.get(sys.platform, 0)
```

Comment:
@st-pasha Do you think we also want to add ppc tolerance here? Right now I see only one test that is disabled in ppc due to the loss of precision.

Reply:
sure

Reply:
Looks like it is not fun to detect ppc https://github.com/h2oai/datatable/blob/master/tests/conftest.py#L33-L47 :)